src/node/src/main.rs

```diff
@@ -36,12 +37,10 @@ fn validator_process() {
     let peer_id = PeerId::from_public_key(pk);
     info!("peer id = {:?}", peer_id);
     let mut swarm = SimpleSwarm::new();
-    for each in NODE_MSG_TOPIC_STR {
-        swarm.topic_list.push(String::from(each.clone()));
-    }
-    for each in CONSENSUS_MSG_TOPIC_STR {
-        swarm.topic_list.push(String::from(each.clone()));
-    }
+    swarm
+        .topic_list
+        .push(String::from(constants::CONSENSUS.clone()));
```

Comment:
the objective of using NODE_MSG_TOPIC_STR static array is, if a new topic is to be added, it just required addition in array and no other code change.
This objective is defeated now !

Reply:
Assuming, not more than 2 topics will be there !